tests/integration/extension/src/background.js

```diff
@@ -44,7 +44,7 @@ browser.scripting.registerContentScripts([{
     id: "webextension-test",
     js: ["dist/browser-polyfill.min.js", "dist/test.content.js"],
     matches: ["<all_urls>"],
-    persistAcrossSessions: false,
+    persistAcrossSessions: true,
```

Comment:
We should add the same check/unregistering before registering the content script here, for posterity (even though it's not going to make a difference for now, we'll probably want to explicitly add tests for this sort of stuff soon anyway)